content/en/platform/corda/5.2/developing-applications/cordapp-template/running-your-first-cordapp/_index.md

```diff
@@ -66,7 +68,7 @@ Once authorised, you can start hitting endpoints. The easiest one to try is `/cp
    As we have not uploaded any CPIs yet, the returned list of CPIs is empty.
    If Corda has not started yet, Swagger will return an error:
    {{< figure src="get-cpi-error.png" figcaption="Swagger showing an error response to GET /cpi" alt="Swagger showing an error response to GET /cpi" >}}
-   If this occurs, you either have not started Corda, Corda has not finished starting, or something has gone wrong. If something has gone wrong, you should try again or [reset the environment]({{< relref "../reset-csde.md" >}}) and start again.
+   If this occurs, you either have not started Corda, Corda has not finished starting, or something has gone wrong. If something has gone wrong, you should try again or [reset the environment]({{< relref "../reset.md" >}}) and start again.
```

Comment:
Line 81: CSDE mentioned in `figcaption` and `alt` of the vNodeSetup image

Reply:
Line 80: replace `5-vNodeSetup` with `vNodesSetup`

Reply:
Should this be renamed throughout? What about `4-deployCPIs` etc?

Reply:
yes, most of the tasks were renamed